Assets/Scenes/GameBoardMain/Scripts/GetCoordinates.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace CoordinateSystem
+{
+    class GetCoordinates
+    {
+        public static List<List<Array>> Coordinates (int rows, int cols, int max)
```

Comment:
Working with jagged arrays in C# is much more challenging than in JS. I opted to use Lists instead. I can explain this reasoning to you if you like.